appstorereviews.js

```diff
@@ -198,14 +198,15 @@ var slackMessage = function (review, config, appInformation) {
     return {
         "username": config.botUsername,
         "icon_url": config.botIcon,
+        "icon_emoji": config.botEmoji,
```

Comment:
Where does the emoji show up in slack when using this?

Reply:
I removed the text from here b/c it was relevant to our app, but here is a sample. In my case I am using 🤖 for Android reviews and 🍎 for iOS reviews:

<img width="341" alt="screen shot 2018-04-20 at 12 43 13 pm" src="https://user-images.githubusercontent.com/1313555/39072244-cc27dba4-449e-11e8-97a2-d586a4f3e788.png">

Reply:
Also in Slack these correspond to `:apple:` and `:robot_face:`:

```javascript
reviews.start({
    apps: [
      {
        appId: "com.app",
        botUsername: "Android Reviews",
        publisherKey: "/path/to/publisher.json",
        channel: '#mobile-app-reviews',
        botEmoji: ":robot_face:"
      },
      {
        appId: "app-id",
        botUsername: "iOS Reviews",
        channel: '#mobile-app-reviews',
        botEmoji: ":apple:"
      }
    ],
    slackHook: 'https://hooks.slack.com/...',
    ...
});
```

Reply:
Ah that's great!